### ISSUE TITLE
Parenthesis content in file name ignored when scanning 

### ISSUE THREAD

**Issue by dertasiu** [Kareadita/Kavita#847]
**If this is a feature request, request [here](https://feats.kavitareader.com/) instead. Feature requests will be deleted from Github.**

Please put as much information as possible to help me understand your issue. OS, browser, version are very important!

**Describe the bug**
There are some spin-off series that have a word in parentheses in the title to disambiguate from the main series.
If the scanner finds one file with parenthesis, it ignores the content and leaves the series' title without that part.

**To Reproduce**
1. Create a file named 'Test manga - Ch 01.cbz', another one named 'Test manga - Ch 02.cbz', and another one named 'Test (test) manga - Ch 03.cbz'
2. Click on 'Scan Series' or 'Scan Library'
3. Instead of two different series, now there is only one without the content of the parenthesis.

**Expected behavior**
It shouldn't ignore the content of the parenthesis and create two different series instead.

**Screenshots**
![parenthesis](https://user-images.githubusercontent.com/11269785/145689671-79e49fca-3032-4a2b-a37d-adcc34c3db80.png)

**Desktop (please complete the following information):**
 - OS: Docker
 - Browser: Firefox 95.0
 - Version: 0.4.9.15

**Additional context**
Ignore the "CANNOT READ" error. The test files are 0kb in size just to recreate the error.

**Comment by majora2007** [comment]
This is a tricky one. I know that usually () have non-series information in them. Do you have an example where the series name has () in it? 

Also, in the next version, local metadata is integrated, so you could instead put a ComicInfo.xml with the series name you want and not worry about the filename.

**Comment by dertasiu** [comment]
Yeah, I know it's tricky hahaha.

Well, the thing is that I was tagging my manga with Calibre and embedding the metadata into a ComicInfo.xml file with a plugin so Kavita could read the metadata.
Since Kavita is awesome, it allows me to have the volumes in CBZ with the ComicInfo tags and in another directory, CBZs of individual chapters that haven't appeared yet in any volume.
When I was tagging this manga, Kavita created a series with the ( ) with the volumes that were tagged, and another series without it, containing only the chapters.

I didn't have any problem with the other series, just in one of them where I had to change the name to the japanese name but romanized so it could merge with the rest of the series.
It would be a pain to tag every individual "temporary" chapter in CBZ with the metadata because I will merge them into a volume once it's completed.

**Comment by dertasiu** [comment]
That's a great idea and I might use it in another series when I encounter a similar problem.
But this case is one that is a little special, I will exemplify-it with my current situation.
I have "3" series:
- The main one with the "English release" of the manga named with a localized name in english. "English Manga - Vol. 1.cbz"
- The main series but in this case, it is a "fan-translated release" that has more volumes. I've named this release with the romanized Japanese name. "Japanese Manga - Vol. 1.cbz"
- And the spinoff that doesn't have a release outside Japan, so it is a fan-translated release and has a romanized Japanese name with the parenthesis. "Japanese (not) Manga - Vol. 1.cbz"

When Kavita scans the Japanese main series, it creates the series "Japanese Manga" and puts in there all the volumes.
But when Kavita scans the spinoff series it deletes all that there is in within the parenthesis and it merges the release with the original series. "Japanese (not) Manga" -> "Japanese Manga"
If only the scanner would leave the "not" there as a part of the title without the parenthesis, I could do what you just told me, set the "Localized name" to "Japanese not Manga" and Kavita would merge them.

I don't know if it is a setting in the Parser when it Normalizes the name or if it is too complicated to change.
If it is a case that and it's only happening to me, I can change the names of the individual chapters and replace the parenthesis with an underline and fix it with the fix that you proposed, hahaha.

**Comment by majora2007** [comment]
Sorry it's been so long since I replied, I went on holiday for 2 weeks. Kavita currently will cleanup anything in (), {}, [], or (complete) from the series title. As you suggested, the key would be to have some sort of marker that doesn't get cleaned, like "Japanese Name NR" (NR for Not Released) or underlines. 

There is no settings for Normalization and it's not something that can be changed unfortunately.
